src/renderer/features/contacts/ContactFilter/model/__tests__/contact-filter.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { fork } from 'effector';
+
+import { filterModel } from '@features/contacts';
+import { contactModel } from '@entities/contact';
+
+const mockContact1 = {
+  id: 1,
+  name: 'name 1',
+  address: '222',
+  accountId: '0x333',
+  matrixId: '@444',
+};
+
+const mockContact2 = {
+  id: 5,
+  name: 'name 6',
+  address: '777',
+  accountId: '0x888',
+  matrixId: '@999',
+};
+const getScopeByQuery = (query: string) => {
+  return fork({
+    values: new Map().set(contactModel.$contacts, [mockContact1, mockContact2]).set(filterModel.$filterQuery, query),
+  });
+};
+
+describe('features/contacts/model/contact-filter-model', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
`jest.clearAllMocks` is redundant here (this seems to be a mistake, because we should use `jest.restoreAllMocks` to restore spies - [doc](https://jestjs.io/docs/jest-object#jestrestoreallmocks))
Following code doesn't contain any spies or whatever mocks

Reply:
Thanks, that's the result of copy/pasting from other tests I found in the code.. and I can't say I understand 100% :)